CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release 0.16.0-dev
+# Release 0.16.0
 
 ## New features
```

Comment:
There seem to be a couple of sections that are empty and we'd like to be removed here: `## Improvements`, `## Breaking changes`, `## Documentation`.